test/models/location_test.rb

```diff
@@ -2,11 +2,7 @@
 
 class LocationTest < ActiveSupport::TestCase
   context "validations" do
-    should validate_presence_of(:country).with_message("Please enter a country")
-    should validate_length_of(:country).is_at_most(50).with_message("50 characters maximum")
```

Comment:
Ugh. Shoulda matchers don't have a built-in way to handle validations with conditionals. I'll just remove these unless anyone has objections and wants to write stubs for these.